geocoordinates/geocoordinates.php

```diff
@@ -24,36 +26,39 @@ function geocoordinates_uninstall()
 	Addon::unregisterHook('post_remote',    'addon/geocoordinates/geocoordinates.php', 'geocoordinates_post_hook');
 }
 
-function geocoordinates_resolve_item(&$item)
+function geocoordinates_resolve_item(array &$item)
 {
-	if((!$item["coord"]) || ($item["location"]))
+	if (empty($item['coord']) || !empty($item['location'])) {
 		return;
+	}
 
 	$key = Config::get("geocoordinates", "api_key");
-	if ($key == "")
+
+	if ($key == "") {
 		return;
+	}
 
-	$language = Config::get("geocoordinates", "language");
-	if ($language == "")
-		$language = "de";
+	$language = Config::get("geocoordinates", "language", "de");
 
 	$coords = explode(' ',$item["coord"]);
 
-	if (count($coords) < 2)
+	if (count($coords) < 2) {
 		return;
+	}
 
 	$coords[0] = round($coords[0], 5);
 	$coords[1] = round($coords[1], 5);
 
 	$result = Cache::get("geocoordinates:".$language.":".$coords[0]."-".$coords[1]);
+
 	if (!is_null($result)) {
 		$item["location"] = $result;
 		return;
 	}
 
 	$s = Network::fetchUrl("https://api.opencagedata.com/geocode/v1/json?q=".$coords[0].",".$coords[1]."&key=".$key."&language=".$language);
 
-	if (!$s) {
+	if ($s === FALSE || empty($s)) {
```

Comment:
This can be simplified to `if (empty($s)) {` but the whole change wasn't warranted either.